DSharpPlus/DiscordConfiguration.cs

```diff
@@ -111,6 +111,7 @@ public string Token
         /// <summary>
         /// <para>Sets the size of the global message cache.</para>
         /// <para>Setting this to 0 will disable message caching entirely. Defaults to 1024.</para>
+        /// <para>This is only applied if the default message cache implementation is used.</para>
```

Comment:
we should consider exposing the config to custom providers, should they chooseto respect it

Reply:
Hmm, you mean we can let this `get` be public?

Reply:
Although on second thought it also seems weird to let custom providers access this from the configuration object. Since the library now doesn't give any way for external code to retrieve it, it will still have to be passed from the user's side. 🤔